File: src/com/leetcode/Easy/FindTheHighestAltitude.java
package com.leetcode.Easy;

// 1732. Find the Highest Altitude
public class FindTheHighestAltitude {
    public int largestAltitude(int[] gain) {
        int max = 0, maxAlt = 0;
        for (int netGain : gain) {
            max += netGain;
            maxAlt = Math.max(maxAlt, max);
        }

        return maxAlt;
    }
}

File: JavaScript/Easy/FindTheHighestAltitude.js
/**
 * 1732. Find the Highest Altitude
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function (gain) {
  let max = 0, maxAlt = 0;
  for (let i = 0; i < gain.length; i++) {
    max += gain[i];
    maxAlt = Math.max(maxAlt, max);
  }
  return maxAlt;
};
